cypress/e2e/tests/migration/task-manager/task_manager_drawer.test.ts

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    deleteApplicationTableRows,
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+} from "../../../../utils/utils";
+import { Analysis } from "../../../models/migration/applicationinventory/analysis";
+import { taskNotificationBadge } from "../../../views/common.view";
+
+const analyses: Analysis[] = [];
+const NUMBER_OF_APPS = 2;
+
+function validateTasks(taskType: string, expectedTasks: string[]) {
+    cy.get("h2.pf-v5-c-notification-drawer__list-item-header-title").each((item) => {
```

Comment:
Could you please refer to a descriptive variable instead of `"h2.pf-v5-c-notification-drawer__list-item-header-title"`?